.travis.yml

```diff
@@ -25,10 +25,6 @@ install:
   - pip3 install -r requirements.dev.txt
   - pip3 install coverage
   - pip3 install codecov
-  - pip3 install gunicorn
```

Comment:
ah! good catch!

Reply:
Hey, why do we still have `.travis.yml` at all? We don't use Travis anymore...